test/build.gradle

```diff
@@ -20,7 +20,7 @@ dependencies {
 sourceSets {
     test {
         resources {
-            srcDir '../prod' // Add pizza shop to resources for selenium tests
+            srcDir '../pizza-shop' // Add pizza shop to resources for selenium tests
```

Comment:
I still think, we should move that to src/main/resources/pizza-shop - then this special resources config would not be needed, and this looks like standard.

Reply:
but nevertheless I will merge that.